go/vt/mysqlctl/builtinbackupengine.go

```diff
@@ -446,13 +535,13 @@ func (be *BuiltinBackupEngine) backupFile(ctx context.Context, params BackupPara
 		}
 	}
 
-	// Flush the buffer to finish writing on destination.
-	if err = dst.Flush(); err != nil {
+	// Close the backupPipe to finish writing on destination.
+	if err = bw.Close(); err != nil {
 		return vterrors.Wrapf(err, "cannot flush destination: %v", name)
 	}
```

Comment:
This is what I was referring to.
The comment says `Close the backupPipe` and the error message says `cannot flush destination`.
I _want_ to approve and merge this PR!
I'll approve provisionally and you can fix the error message and merge it.